extensions/mssql/package.json

```diff
@@ -371,6 +371,11 @@
           "type": "boolean",
           "default": false,
           "description": "%mssql.objectExplorer.groupBySchema%"
+        },
+        "mssql.objectExplorer.expandTimeout": {
+          "type": "number",
+          "default": 45,
+          "description": "%mssql.objectExplorer.expandTimeout%"
```

Comment:
Is this already set up to be consumed in STS?

Reply:
Guess so, @llali I don't suppose you know why this doesn't appear to exist? Did it never get added on the ADS side?

https://github.com/microsoft/sqltoolsservice/pull/363

Reply:
Yeah, OE setting is already using this in STS. https://github.com/microsoft/sqltoolsservice/blob/c26a2aea142524c9dec4625448fe0f1ff2b2fcde/src/Microsoft.SqlTools.ServiceLayer/SqlContext/ObjectExplorerSettings.cs#L34